src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/riscv64/eltwise.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "custom/single_layer_tests/classes/eltwise.hpp"
+#include "utils/cpu_test_utils.hpp"
+#include "utils/fusing_test_utils.hpp"
+#include "utils/filter_cpu_info.hpp"
+#include "nodes/kernels/riscv64/cpu_isa_traits.hpp"
+
+using namespace CPUTestUtils;
+
+namespace ov {
+namespace test {
+namespace Eltwise {
+namespace {
+
+/*
+ * The motivation of this test is to validate different input and output precisions of Eltwise Op.
+ * If IO data type is not supported by jit emitter, they should be converted
+ * to supported types on operation inputs and outputs in JIT kernel
+*/
+
+static const std::vector<ov::test::utils::EltwiseTypes> ops() {
+    // JIT us supported only when `gcv` is available
+    if (ov::intel_cpu::riscv64::mayiuse(ov::intel_cpu::riscv64::gcv)) {
+        return { utils::EltwiseTypes::ADD };
```

Comment:
Sould like we need to cover all supported ops

Reply:
Fair enough.
Then lets check https://github.com/openvinotoolkit/openvino/blob/6805e6229add575fdae0f4a5d90e70713924bbfe/src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp and remove Eltwise tests from the skip. I would expect things like EltwiseChain should be functional now.

Reply:
I enabled a few `Eltwise` tests with integer division now. `EltwiseChain` have been already enabled. 

Thanks for the idea!